pkg/command/domain/list.go

```diff
@@ -0,0 +1,60 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package domain
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/client-contrib/plugins/admin/pkg"
+	"knative.dev/client/pkg/kn/commands/flags"
+)
+
+// NewDomainListCommand represents 'kn-admin domain list' command
+func NewDomainListCommand(p *pkg.AdminParams) *cobra.Command {
+
+	domainListFlags := flags.NewListPrintFlags(DomainListHandlers)
+	domainListCommand := &cobra.Command{
+		Use:   "list",
+		Short: "List custom domain",
```

Comment:
Same as above, I think we can use domain instead of custom domain here.

Reply:
How about this one, what do you think? Thanks!

Reply:
I already made the change. Seems it is outdated. Please check "Files changed" tab. Thank you.